Client/Experiments/Experiments.swift

```diff
@@ -136,10 +136,16 @@ enum Experiments {
 
         let isPhone = UIDevice.current.userInterfaceIdiom == .phone
 
+        var isReviewCheckerEnabled = false
+        if let prefs = UserDefaults(suiteName: AppInfo.sharedContainerIdentifier) {
```

Comment:
Is the defaults firefox-ios uses for all preferences now?

Reply:
Great question, @thatswinnie left for vacation but I wonder why the `suiteName` was used here. We use the suite name in older part of the code, but I'm unsure if there's any reason why this was used here.